app/views/problems/_fields.html.erb

```diff
@@ -20,6 +20,10 @@ allow a student to see their total score without having a score for that
 problem. Use this option carefully. In most cases, you should leave it
 unchecked." %>
 
+<%= f.check_box :starred, help_text: "By default, all problems are
```

Comment:
Would recommend elaborating further on this.

e.g. "Starred problems are displayed first in the problems dropdown when creating annotations"

Reply:
Rephrased in recent commit.